gui-apps/grim/grim-9999.ebuild

```diff
@@ -10,10 +10,11 @@ HOMEPAGE="https://github.com/emersion/grim"
 
 if [[ ${PV} == 9999 ]]; then
 	inherit git-r3
-	EGIT_REPO_URI="https://github.com/emersion/${PN}.git"
+	EGIT_REPO_URI="https://git.sr.ht/~emersion/${PN}"
 else
-	SRC_URI="https://github.com/emersion/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	SRC_URI="https://git.sr.ht/~emersion/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
 	KEYWORDS="~amd64 ~arm64 ~loong ~ppc64 ~riscv ~x86"
+	S="${WORKDIR}/${PN}-v${PV}"
```

Comment:
Here it is fine to change (think of it as template for future ebuild on version bump) and the `S=` is excellent as long as it would be correct (simple to validate on future bump).

Reply:
> Here it is fine to change (think of it as template for future ebuild on version bump) and the `S=` is excellent as long as it would be correct (simple to validate on future bump).

I'll comment it out since it won't work with the current SRC_URI. Thanks!